api/application_test.go

```diff
@@ -155,7 +161,9 @@ func (s *ApplicationSuite) Test_CreateApplication_withExistingToken() {
 
 	expected := &model.Application{ID: 2, Token: secondApplicationToken, Name: "custom_name", UserID: 5}
 	assert.Equal(s.T(), 200, s.recorder.Code)
-	assert.Contains(s.T(), s.db.GetApplicationsByUser(5), expected)
+	if app, err := s.db.GetApplicationsByUser(5); assert.NoError(s.T(), err) {
```

Comment:
The additional ifs let gocyclo fail right? It could be written like this, then it would have a similar effect but no ifs.
```go
app, err := s.db.GetApplicationsByUser(5)
require.NoError(s.T(), err)
assert.Contains(s.T(), app, expected)
```
Tho, I think we do not need this check in most of the tests. In the tests the returned value is equal checked against an expected value and this check would already fail, if the db threw some error. What do you think?

Reply:
If we do not stop the further assertion, a panic might happen and the detailed failure information might be lost.

Reply:
See `require` https://godoc.org/github.com/stretchr/testify/require
> The `require` package have same global functions as in the `assert` package, but instead of returning a boolean result they call `t.FailNow()`.

So after a failure no further assertions will be done.